pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -681,11 +666,6 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			newDisk.Driver.IOThread = &ioThreadId
 		}
 
-		err = setDriverCacheMode(&newDisk, disk.Cache)
```

Comment:
this has been moved to `preStartHook` [link](https://github.com/kubevirt/kubevirt/pull/1651/files#diff-9b1d64950edb57271a1a2fc36192d464R359) after disks initialization.